src/test/java/seedu/recipe/logic/parser/AddCommandParserTest.java

```diff
@@ -1,8 +1,10 @@
 package seedu.recipe.logic.parser;
 
 import static seedu.recipe.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+import static seedu.recipe.logic.commands.CommandTestUtil.*;
```

Comment:
*-import here

Reply:
Ok, I changed my code style to not do that anymore. Sorry about that.